JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -71,7 +71,7 @@ module Microsoft.ApplicationInsights.Context {
                 this.initializeAutomaticSession();
             }
 
-            var now = +new Date;
+            var now = dateTime.Now();
```

Comment:
K, the comment on that method isn't accurate. You may want to correct that. performance.now is the time since the context was created and not the time since 1970.